src/main/java/no/nav/pto/veilarbportefolje/oppfolging/OppfolgingService.java

```diff
@@ -59,103 +67,126 @@ public void lastInnDataPaNytt() {
         JobRunner.runAsync("OppfolgingSync",
                 () -> {
                     antallBrukereSlettet = 0;
-                    List<OppfolgingsBruker> oppfolgingsBruker = brukerRepository.hentAlleBrukereUnderOppfolging();
+                    List<AktorId> oppfolgingsBruker = oppfolgingRepository.hentAlleBrukereUnderOppfolging();
                     oppfolgingsBruker.forEach(this::oppdaterBruker);
 
                     log.info("OppfolgingsJobb: oppdaterte informasjon pa: {} brukere der av: {} ble slettet", oppfolgingsBruker.size(), antallBrukereSlettet);
                 });
     }
 
-    public void oppdaterBruker(OppfolgingsBruker bruker) {
-        if (bruker.getAktoer_id() == null) {
-            return;
-        }
-        if (bruker.getFnr() == null) {
-            log.error("Fnr var null pa bruker: " + bruker.getAktoer_id());
+    public void oppdaterBruker(AktorId bruker) {
+        if (bruker == null) {
             return;
         }
 
         try {
-            Optional<OppfolgingPeriodeDTO> oppfolgingPeriode = hentSisteOppfolgingsPeriode(bruker.getFnr());
-            if (oppfolgingPeriode.isPresent()) {
-                oppdaterStartDato(bruker, oppfolgingPeriode.get().startDato);
-                avsluttOppfolgingHvisNodvendig(bruker, oppfolgingPeriode.get());
+            Veilarbportefoljeinfo veialrbinfo = hentVeilarbData(bruker);
+            if (veialrbinfo.isErUnderOppfolging()) {
+                Optional<BrukerOppdatertInformasjon> dbInfoOracle = oppfolgingRepository.hentOppfolgingData(bruker);
+                Optional<BrukerOppdatertInformasjon> dbInfoPostgres = oppfolgingRepositoryV2.hentOppfolgingData(bruker);
+
+                oppdaterStartDatoHvisNodvendig(bruker, dbInfoOracle.map(BrukerOppdatertInformasjon::getStartDato).orElse(null), veialrbinfo.getStartDato(), false);
+                oppdaterStartDatoHvisNodvendig(bruker, dbInfoPostgres.map(BrukerOppdatertInformasjon::getStartDato).orElse(null), veialrbinfo.getStartDato(), true);
+
+                oppdaterManuellHvisNodvendig(bruker, dbInfoOracle.map(BrukerOppdatertInformasjon::getManuell).orElse(false), veialrbinfo.isErManuell(), false);
+                oppdaterManuellHvisNodvendig(bruker, dbInfoOracle.map(BrukerOppdatertInformasjon::getManuell).orElse(false), veialrbinfo.isErManuell(), true);
+
+                oppdaterNyForVeilederHvisNodvendig(bruker, dbInfoOracle.map(BrukerOppdatertInformasjon::getNyForVeileder).orElse(false), veialrbinfo.isNyForVeileder(), false);
+                oppdaterNyForVeilederHvisNodvendig(bruker, dbInfoOracle.map(BrukerOppdatertInformasjon::getNyForVeileder).orElse(false), veialrbinfo.isNyForVeileder(), true);
+
+                oppdaterVeilederHvisNodvendig(bruker, dbInfoOracle.map(BrukerOppdatertInformasjon::getVeileder).orElse(null), Optional.ofNullable(veialrbinfo.getVeilederId()).map(NavIdent::get).orElse(null), false);
```

Comment:
can you call call once this function...seems like `postgres` parameter doesnt metter much since it has both values

Reply:
```
if (postgres) {
            log.info("(Postgres) OppfolgingsJobb: aktoer: {} skal bytte veileder fra: {}, til:{} ", bruker, veilederDb, korrektVeileder);
            oppfolgingRepositoryV2.settVeileder(bruker, VeilederId.of(korrektVeileder));
        } else {
            log.info("OppfolgingsJobb: aktoer: {} skal bytte veileder fra: {}, til:{} ", bruker, veilederDb, korrektVeileder);
            oppfolgingRepository.settVeileder(bruker, VeilederId.of(korrektVeileder));
        }
```

-->
```
 log.info("(Postgres) OppfolgingsJobb: aktoer: {} skal bytte veileder fra: {}, til:{} ", bruker, veilederDb, korrektVeileder);
            oppfolgingRepositoryV2.settVeileder(bruker, VeilederId.of(korrektVeileder));
log.info("OppfolgingsJobb: aktoer: {} skal bytte veileder fra: {}, til:{} ", bruker, veilederDb, korrektVeileder);
            oppfolgingRepository.settVeileder(bruker, VeilederId.of(korrektVeileder));
```

Reply:
But there are different inputs for postgres vs oracle 🤔

Reply:
oops!

Reply:
I will write a test to see if everything works in migration to postgres